http/1.6/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -581,63 +545,50 @@ This example includes two `oauth:custom-parameter` child elements that define pa
 </http:request-config>
 ----
 
-////
-=====
-////
-
-=== Override the Redirect URI
-
-This section lets you override the redirect URI (external `redirect_uri`).
+=== Configure OAuth2 Authorization Redirect URI
 
 The https://tools.ietf.org/html/rfc6749[OAuth 2.0 specification] describes checking the redirect URI from the destination site of the redirect. The OAuth authentication server uses the URL to provide the authentication code to the Mule server for retrieving the access token. If you provide this URL, Mule creates an endpoint at the URL for storing the authentication code unless there’s already an endpoint registered to manually extract the authorization code.
 
-You configure the external redirect URI by setting the External Callback URL attribute (`externalCallbackUrl` in XML).
-
-Using `externalCallbackUrl` is particularly useful for deploying applications to CloudHub, for example. When configuring authentication, you can optionally specify the `localCallbackUrl` attribute as well.
+To override the redirect URI (external `redirect_uri`) configure the external redirect URI in the *External callback url* field, which is useful for deploying applications to CloudHub, for example. When you configure authentication, you can optionally configure the *Local callback url* field as well.
 
-For example, the `localCallbackUrl` is `+http://localhost:8082/callback+` in the <<create-the-mule-client-app,previous example>>:
-
-To create the endpoint for CloudHub, Mule has to create an endpoint for CloudHub in a different format. For example:
+To create the endpoint for CloudHub, Mule has to create an endpoint for CloudHub in a different format, for example:
```

Comment:
I don't understand this sentence: "to create the endpoint" Mule has to "create an endpoint"?
And "in a different format"? different from what?